index.html

```diff
@@ -719,6 +720,16 @@ <h3>
           "FETCH#concept-filtered-response-opaque">opaque filtered
           response</a>.
         </p>
+        <p data-dfn-for="PerformanceResourceTiming">
+          The <dfn>contentType</dfn> getter steps are to return
+          <a>this</a>'s <a data-for="PerformanceResourceTiming">resource
+          info</a>'s [=response body info/content type=]'s <a data-cite=
+          "MIMESNIFF#mime-type-essence">essence</a> if
+          <a>this</a>'s <a data-for=
+          "PerformanceResourceTiming">resource info</a>'s
+          [=response body info/content type=] is non-null; otherwise
+          the empty string.
+        </p>
```

Comment:
Rather than just returning the essence, this should do https://mimesniff.spec.whatwg.org/#serializing-a-mime-type, no?

Reply:
Yeah, that seems more appropriate. Updated.